File: swing/src/main/java/wannabe/swing/renderer/Circle.java
package wannabe.swing.renderer;

import java.awt.Graphics;
import wannabe.swing.SwingProjected;

public class Circle extends SwingRenderer {
  @Override public void draw(Graphics g, SwingProjected p) {
    g.drawOval(p.left, p.top, p.size, p.size);
  }
}

File: swing/src/main/java/wannabe/swing/renderer/Circle.kt
package wannabe.swing.renderer

import wannabe.swing.SwingProjected
import java.awt.Graphics

class Circle : SwingRenderer() {
  override fun draw(
    g: Graphics,
    p: SwingProjected
  ) {
    g.drawOval(p.left, p.top, p.size, p.size)
  }
}
